File: api-server/src/main/java/com/talentlink/talentlink/favorite/dto/FavoriteResponse.java
package com.talentlink.talentlink.favorite.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.LocalDateTime;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class FavoriteResponse {

    private Long id;
    private String userId;
    private String type;
    private Long sellId;   // 판매글일 때 해당 게시글 id
    private Long buyId;    // 구매글일 때 해당 게시글 id
    private String title;
    private String writerNickname;
    private LocalDateTime time;

}

File: front-server/app/src/main/java/com/example/talent_link/ui/Favorite/dto/FavoriteResponse.kt
package com.example.talent_link.ui.Favorite.dto

data class FavoriteResponse(
    val id: Long,
    val userId: String,
    val type: String,
    val title: String,
    val writerNickname: String,
    val sellId: Long? = null,   // 추가!
    val buyId: Long? = null,    // 추가!
    val time: String? = null
)
